### ISSUE TITLE
rdap-srv-store zeros rdap json files

### ISSUE THREAD

**Issue by jumpifnotzero** [icann/icann-rdap#12]
I created one object with `rdap-srv-data network --cidr 224/8`

```
# ls -l /tmp/rdap-srv/data/
-rw-r--r-- 1 root root 564 Jun  6 00:44 localhost_3000_rdap_ip_224_0_0_0_8.json
```
I then ran `rdap-srv-store /tmp/rdap-srv/data` which zero'd out the file.

I guess this is because the source and destination directories to rdap-srv-store are the same?! I understand I should be managing the rdap "source" files in another directory and using rdap-srv-store to copy them over.

**Comment by anewton1998** [comment]
This is will be fixed when commit ff9d0dc546b73d0336b77327b132e89b366e8fe0 is merged.

**Comment by anewton1998** [comment]
Merged in #20
